explainer.md

```diff
@@ -119,131 +119,129 @@ function BeginVRSession(isExclusive) {
+
+When `VRWebGLLayer.multiview` is true:
+- The UA may decide to back the framebuffer with a texture array, side-by-side texture or another implementation of the UA's choosing. This implementation decision must not have any impact how developers author their shaders or setup the WebGL context for rendering. 
+- When calling `VRPresentationFrame.getViewport()` with this type of `VRWebGLLayer`, the `VRView` parameter will be ignored.  All `VRView`s will have the same `VRWebGLViewport`
+
+When `VRWebGLLayer.multiview` is false:
+- The `VRWebGLLayer` is created with a context that does not support this extension, the framebuffer must be created in a side-by-side configuration.
+- When calling `VRPresentationFrame.getViewport()` with this type of `VRWebGLLayer`, a valid `VRView` parameter is required and will return a different `VRWebGLViewport` for each.
 
 ```js
-// Initially only canvases with WebGL contexts will be supported.
 let glCanvas = document.createElement("canvas");
 let gl = glCanvas.getContext("webgl");
 
 let vrSession = null;
-let vrLayer = null;
 
 function OnSessionStarted(session) {
   // Store the session for use later.
   vrSession = session;
-
-  if (vrSession.createParameters.exclusive) {
-    // In order to ensure the canvas renders at an appropriate size for the
-    // device the preferred canvas size should be queried from the session.
-    let sourceProperties = vrSession.getSourceProperties();
-    glCanvas.width = sourceProperties.width;
-    glCanvas.height = sourceProperties.height;
-  }
-
+  
   // The depth range of the scene should be set so that the projection
   // matrices returned by the session are correct.
   vrSession.depthNear = 0.1;
   vrSession.depthFar = 100.0;
 
-  // The content that will be shown on the device is defined by the session's
-  // baseLayer. In non-exclusive The baseLayer is not used for presentation, but
-  // the canvas dimensions are used to construct the projection matrices.
-  vrLayer = new VRCanvasLayer(vrSession, glCanvas);
-  vrSession.baseLayer = vrLayer;
+  // The content that will be shown on the device is
+  // defined by the current layer.
+  // VRWebGLLayer allows for the optional use of the WEBGL_multiview extension
+  vrSession.baseLayer = new VRWebGLLayer(vrSession, glCanvas, { multiview: true });
 
   // Start the render loop
-  vrSession.commit().then(OnFirstVRFrame);
+  vrSession.requestVRPresentationFrame().then(onDrawFrame);
```

Comment:
Not a big deal, but is there a reason for preferring `VRPresentationFrame` instead of just `VRFrame`? I'm not overtly concerned about verbosity, but it's nice to cut down on it when reasonable.